src/test/java/teammates/test/cases/action/AdminActivityLogPageActionTest.java

```diff
@@ -381,53 +379,52 @@ public void removeAndRestoreManyLogs() {
         gaeSimulation.loginAsAdmin("admin");
         gaeSimulation.clearLogs();
 
-        Date today = TimeHelper.getDateOffsetToCurrentTime(0);
+        Instant today = TimeHelper.getInstantDaysOffsetFromNow(0);
         insertLogMessageAtTimeWithInterval(logMessages.get(LOG_MESSAGE_INDEX_MANY_LOGS),
-                today.getTime(), LOG_MESSAGE_INTERVAL_MANY_LOGS);
+                today.toEpochMilli(), LOG_MESSAGE_INTERVAL_MANY_LOGS);
     }
 
     // The two test groups should have different 'priority' so that they can run separately
     // as they depend on different sets of log messages
     @Test(groups = "manyActivityLogs", priority = 2)
     public void statusMessageAndContinueSearch_withManyLogs_searchCorrectly() {
-        Date today = TimeHelper.getDateOffsetToCurrentTime(0);
-
+        Instant today = TimeHelper.getInstantDaysOffsetFromNow(0);
```

Comment:
I didn't use the member variable here because this is necessary for the logic of the test. Basically the test expect this `today` to be after the `today` in the previous function, because searching is not inclusive, so having this `today` after the log's `today` kind of make the search as if it's inclusive.

Reply:
Can rename this variable to `now` and use `Instant.now()`?